classes/01class/exercises/c01-aws01/nimmi89/README.md

```diff
@@ -21,10 +21,10 @@ aws ec2 create-security-group \
     "GroupId": "sg-0619b1745d16f5cb6"
 }
 
->>>Command to add a rule in that security group that allows SSH access from anywhere.
+>>>Command to add a rule in that security group that allows SSH access from a specific IP address only(enhanced security)
 aws ec2 authorize-security-group-ingress \
 --group-id sg-0619b1745d16f5cb6 \
---protocol tcp --port 22 --cidr 0.0.0.0/0
+--protocol tcp --port 22 --cidr 192.168.64.9/32
```

Comment:
are you sure this still works?

Reply:
Have assigned my public IP returned by- curl https://checkip.amazonaws.com!!